modules/logging-otlp-forward.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-output.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-otlp-forward_{context}"]
+= Forwarding logs over OTLP
+
+Forwarding logs over OpenTelemetry Protocol (OTLP) is supported for the Vector log collector. To enable, specify `otlp` as the output type in the `ClusterLogForwarder` custom resource (CR).
+
+.Procedure
+
+* Create or edit the `ClusterLogForwarder` CR using the template below:
++
+.Example ClusterLogForwarder CR
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+    name: my-logforwarder
+    namespace: my-app-namespace
+    annotations:
+        observability.openshift.io/tech-preview-otlp-output: "enabled"
+spec:
+    outputs:
+    - type: otlp <1>
+      name: otel-collector
+      tls:
+        insecureSkipVerify: true <2>
+      otlp:
+        url: 'http://my-otel-receiver-service:4318/v1/logs' <3>
+        tuning:
+            compression: none <4>
+        authentication:
+            token:
+            from: serviceAccount <5>
+    pipelines:
+    - name: my-pipeline
+        inputRefs:
+        - application
+        - infrastructure
+        outputRefs:
+        - otel-collector
+    serviceAccount:
+    name: logger-admin
+----
+<1> The output type value is `otlp`.
+<2> The `tls` includes the standard certificate configuration or set the field `insecureSkipVerify` value to `true`.
```

Comment:
If I recall correctly, there's something in the source noting that this is not a recommended configuration for production, that should be included here.